src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -156,9 +156,9 @@ public void canCreateARequest() {
       representation.getJsonObject("requester").getString("firstName"),
       is("Steven"));
 
-    assertThat("middle name is not taken from requesting user",
-      representation.getJsonObject("requester").containsKey("middleName"),
-      is(false));
+    assertThat("middle name is taken from requesting user",
```

Comment:
Why has this comment been changed to the positive description of the assertion, when the message is reported only in the negative case?

Reply:
the condition was changed as well

Reply:
Ah, I think this is where my confusion came from.

Maybe the message could state something like `middle name from requester should be present`, which explains what the expectation of the failed assertion was?

Reply:
I took the phrasing from adjacent assertions. Should I change them as well?

Reply:
Personally, I think the phrasing of assertion messages should describe how the expectations were not met.

This was only a question, if you want to leave them self-consistent, that is ok, and this can be resolved.

Reply:
Looks like you are the one who created those assertions in the first place, so if you want me to rephrase them, I'm more than happy to :)

Reply:
Yeah, let's leave them. I think I reworked this code heavily from an original that did things a different way